js/apps/admin-ui/src/organizations/OrganizationForm.tsx

```diff
@@ -25,13 +27,19 @@ export const convertToOrg = (
 
 export const OrganizationForm = () => {
   const { t } = useTranslation();
+  const { tab } = useParams<EditOrganizationParams>();
   return (
     <>
       <TextControl
         label={t("name")}
         name="name"
         rules={{ required: t("required") }}
       />
+      <TextControl
+        label={t("alias")}
+        name="alias"
+        readOnly={tab === "settings"}
```

Comment:
@edewit I did not manage to automatically fill the `alias` field when typing the name. Do you have any tip?

Reply:
Hi @pedroigor after creating the organization, can users edit the alias?

Reply:
No, he cannot.

Reply:
Thanks @pedroigor I understood. 
If so, I agree with you. we can make the `Alias` refer to the `Name` by default. after typing the `Name` and losing focus, the `Alias` will be automatically filled in by the `name` as shown in the 1st screenshot. If users leave the `Alias` empty, we can fill in the name after creating the organization.
As for the `Alias`, we could add a tooltip to explain the mechanism as shown in the 2nd screenshot. (The wording needs to be optimized)
After creation, the `Alias` field should be disabled as shown in the 3rd screenshot.
Does it make sense?

> <img width="655" alt="Screenshot 2024-06-12 at 21 38 10" src="https://github.com/keycloak/keycloak/assets/53931519/59a21be0-b357-48a5-aeb2-7da00adaab6e">

> <img width="429" alt="Screenshot 2024-06-12 at 21 39 00" src="https://github.com/keycloak/keycloak/assets/53931519/b4fb3ed3-860e-4d5b-b37e-f68f5f045312">

> <img width="740" alt="Screenshot 2024-06-12 at 21 39 41" src="https://github.com/keycloak/keycloak/assets/53931519/c119444a-924c-4612-9d90-ce4cbb529a2a">

Reply:
Yes, it makes sense. The alias is already read-only once the org is created (editing) I need now to:

* Add tooltip
* Automatically fill when typing the name (or loosing focus)

Thanks!

Reply:
Sounds good. WRT tooltip, the content is my rough idea. Please optimize the wording according to the use case.

Reply:
It sounds great. Thanks for sharing this.